### PR TITLE
ADEN-5710 Renames Rubicon Prebid adapters

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L79](https://github.com/Wikia/app/blob/5406d9a65223c39deb3f4b6591d0e7b81beab410/extensions/wikia/AdEngine/js/context/adContext.js#L79): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L79](https://github.com/Wikia/app/blob/9abd3e82497853226c686a23fc16823da6e6ef63/extensions/wikia/AdEngine/js/context/adContext.js#L79): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/9abd3e82497853226c686a23fc16823da6e6ef63/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1118 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by nandy-andy** [comment]
The code changes look fine but before moving forward I'd suggest happy-path-test two things:
* take a look if [Kikimora still gets correct values](https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js#L61),
* use [Requestly](https://wikia-inc.atlassian.net/wiki/spaces/ADEN/blog/2016/08/01/91324434/Requestly+as+a+cache+tester) and check if a cached content doesn't break anything with new assets and variables as it has cached ads context inlined.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L79](https://github.com/Wikia/app/blob/c45407f0ff66a350f6d38c90f4f08d25308eab3b/extensions/wikia/AdEngine/js/context/adContext.js#L79): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L79](https://github.com/Wikia/app/blob/d834eafe281fe4946756c767518629243a0e98e7/extensions/wikia/AdEngine/js/context/adContext.js#L79): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 			'bidder_8': transformBidderPrice('veles', realSlotPrices, slotPricesIgnoringTimeout),
 			'bidder_9': transformBidderPrice('openx', realSlotPrices, slotPricesIgnoringTimeout),
 			'bidder_10': transformBidderPrice('appnexusAst', realSlotPrices, slotPricesIgnoringTimeout),
+			'bidder_11': transformBidderPrice('rubiconDisplay', realSlotPrices, slotPricesIgnoringTimeout),
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js]: Is kikimora ready for this new field?
https://github.com/Wikia/crossbow/blob/master/dags/winter_wind/analytics-wikia/fastly/special/fact_adengadinfo_events/fact_adengadinfo_events_insert.sql

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L79](https://github.com/Wikia/app/blob/d0c135e6898445e1d117f413592b53fd70346840/extensions/wikia/AdEngine/js/context/adContext.js#L79): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/2652ac9f2fc6222a3d1f9b398f17ae4309cb8811/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/e8ab07c016785d604500c72f0e74436f70135256/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**